sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.FormRecognizer.Training
+{
+    /// <summary>
+    /// The client to use to with the Form Recognizer Azure Cognitive Service to train custom models from forms,
+    /// and to extract values from forms using those custom models.  It also supports listing and deleting trained
+    /// models.
+    /// </summary>
+    public class FormTrainingClient
+    {
+        private readonly ClientDiagnostics _diagnostics;
+        private readonly HttpPipeline _pipeline;
+
+        internal readonly ServiceClient ServiceClient;
+
+        internal const string CustomModelsRoute = "/custom/models";
```

Comment:
I believe this should also be removed because the API paths are handled now in ServiceClient.